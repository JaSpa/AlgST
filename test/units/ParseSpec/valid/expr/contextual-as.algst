-- Check that `as` is a valid variable name.
let as = 10 in as
