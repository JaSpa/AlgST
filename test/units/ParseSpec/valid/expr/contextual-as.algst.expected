Exp.UnLet as
|
+- Exp.Lit
|  |
|  `- Lit.Int 10
|
`- Exp.Var as

